cmd/create/ocmrole/cmd.go

```diff
@@ -101,6 +103,13 @@ func init() {
 		"Attach Classic ROSA AWS managed policies to the account roles. This is an alias for --managed-policies")
 	flags.MarkHidden("mp")
 
+	flags.BoolVar(
```

Comment:
Can we not use the same from the create command?
https://github.com/openshift/rosa/blob/06a141ef9258ebe956428dd1eff33bc4fc88cdc3/cmd/create/ocmrole/cmd.go#L115

I feel like if the user supplies this we can just pass it along

Reply:
There's already a flag for "-y" which gets included through confirm.AddFlag, would it be possible to just pass that value along instead of having a new argument?

Reply:
Okay I think I gt what you mean PTAL